clients/src/main/java/org/apache/kafka/common/errors/DisconnectException.java

```diff
@@ -17,6 +17,7 @@
  * Server disconnected before a request could be completed.
  */
 public class DisconnectException extends RetriableException {
+    public static final DisconnectException INSTANCE = new DisconnectException();
```

Comment:
I meant to use it in ConsumerNetworkClient, but it looks like I left that out. I'll add another commit to fix it.